### PR TITLE
Initial Flow-Enabled Skeleton Commit

### PR DESCRIPTION
**Pull request by dwertheimer**
@nmn Please have a look and see what I got right/wrong. I am still figuring it all out, but I followed your suggestions and I think it's in the neighborhood of correct. @jgclark @EduardMe Please edit as you see fit.

BTW, I did also include a couple of dot-files, because I thought you wouldn't want to have people thinking they need to commit the generated script.js and also VSCode needs some tweaking so it doesn't complain so much about Flow. 

I look forward to your comments/improvements. Thanks @nmn! This is very cool.

### CODE REVIEW

**Comment by dwertheimer** [comment]
BTW, I toyed with the idea of including some helper functions in with the templates (as I think you guys discussed) -- for things we're all using repeatedly, like `dateHelpers.js` and `interpolation.js` and `messageFunctions.js`...would love to hear how you're thinking of dealing with that (and keeping it updated). I know there has been some discussion. :)

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+    "typescript.validate.enable": false,
+    "typescript.suggest.includeAutomaticOptionalChainCompletions": false,
+    "javascript.validate.enable": false
+}
```
Comment (nmn) [inline .vscode/settings.json:5]: Can you remove this from your commit? Perhaps add this file to .gitignore as well.

File: np.plugin-flow-skeleton/rollup.config.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
+// DO NOT EDIT THIS FILE. IT'S FOR THE SCRIPT PACKAGER!
```
Comment (nmn) [inline np.plugin-flow-skeleton/rollup.config.js:1]: You no longer need this file if you just use `nom run autowatch` that one command automatically does the compiling for you.

But feel free to keep it if you think it adds value. I feel like letting one script handles everything automatically is simpler to get started with

**Review by nmn — COMMENTED** [review]
This looks right. Except you accidentally added your .vscode settings.

**Review by nmn — APPROVED** [review]
Approved.

**Comment by dwertheimer** [comment]
I added the .vscode settings because it took me awhile to figure out why
VSCode was yelling at me about Typescript problems (it was trying to read
the Flow), so I figured out how to turn those checks off and I thought I
would include it as a public service, but I will delete it if you think it
streamlines things.

And I will get rid of that script. Good to know it's unnecessary.

Thanks @nmn!

**Review by dwertheimer — COMMENTED** [review]
Made changes recommended by @nmn

**Comment by dwertheimer** [comment]
Also added UI change to the rollup script. @nmn pls make sure you're comfortable with it.

**Comment by dwertheimer** [comment]
@nmn just bumping this

**Comment by jgclark** [comment]
Looks good to me.

**Comment by nmn** [comment]
Yup looks good!

**Comment by dwertheimer** [comment]
Thanks guys. Very cool stuff @nmn. @jgclark u will release it?

**Comment by jgclark** [comment]
If I understand correctly, this doesn't need a *release*, as its not a plugin. It simply needs to be *committed* to main. Unfortunately I can't do that from iPad client, as its telling me (rightly or wrongly) that it cannot automatically be merged. Perhaps @nmn could do so instead?

**Comment by dwertheimer** [comment]
@jgclark I used your .gitignore so there are no merge conflicts. Pls merge this and let's close this PR. 
Thanks!

**Comment by dwertheimer** [comment]
@jgclark @nmn Can you guys pls merge this into main so that I can at least close one of my PRs?
It has been approved by @nmn and has no merge conflicts. Thank you.

**Pull request merged**